snuba/datasets/querylog_processor.py

```diff
@@ -120,9 +120,9 @@ def process_message(
             # TODO: This column is empty for now, we plan to use it soon as we
             # will start to write org IDs into events and allow querying by org.
             "organization": None,
-            "timestamp": message["timing"]["timestamp"],
-            "duration_ms": message["timing"]["duration_ms"],
-            "status": message["status"],
+            "timestamp": message.get("timing", {}).get("timestamp"),
+            "duration_ms": message.get("timing", {}).get("duration_ms"),
+            "status": message.get("status"),
```

Comment:
These three keys have to be missing from the message for clickhouse to add the type default. 

Also please record a metric otherwise we won't know if this is just one broken message or something worse.